### PR TITLE
New package: Wiimm.SZS version 2.40a

### PR DESCRIPTION
**Pull request by Exorcism0666**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132317)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is bot account and belongs to CoolPlayLin. Any issue should call him up instead of assigning to this account. He has the write permission of this account's fork so that he can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let CoolPlayLin know).

Have a good day!

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification. It has the write permission of this account's fork so that it can resolve the issue directly.

>[!important]
>Please recheck these Pull Request carefully before merging. If it is a incorrect-removing Pull Request and the URLs don't have any issue while manual checking, please close this Pull Request directly(It's best to let Exorcism0666 know).

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Response status code does not indicate success: 404 (Not Found).

**Comment by stephengillie** [comment]
200

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 3:57:16 PM: 2024-01-02 15:57:16.218 [CLI ] Package hash verification failed. SHA256 in manifest [7d023b93b806e889152983aa6ee4aa811b7418ea157a0c82732cfd4895f63adb] does not match download [c54738cfcdad74ae49424d4c3f8f16030290045632522263e22e8d7cd6d6ec76]

File: manifests/w/Wiimm/SZS/2.40a/Wiimm.SZS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageIdentifier: Wiimm.SZS
+PackageVersion: 2.40a
+InstallerType: zip
+NestedInstallerType: exe
+Installers:
+- InstallerUrl: https://szs.wiimm.de/download/szs-v2.40a-r8917-cygwin64.zip
+  Architecture: x64
+  NestedInstallerFiles:
+  - RelativeFilePath: szs-v2.40a-r8917-cygwin64\windows-install.exe
+  InstallerSha256: 7D023B93B806E889152983AA6EE4AA811B7418EA157A0C82732CFD4895F63ADB
```
Comment (stephengillie) [inline manifests/w/Wiimm/SZS/2.40a/Wiimm.SZS.installer.yaml:13]:
```suggestion
  InstallerSha256: C54738CFCDAD74AE49424D4C3F8F16030290045632522263E22E8D7CD6D6EC76
```

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Specified hash doesn't match.
Actual hash:   C54738CFCDAD74AE49424D4C3F8F16030290045632522263E22E8D7CD6D6EC76
Expected hash: b1ec14e3295693c167ad0859e75dd14ed1b71aae71334eef843a6c59d4783bf7

File: manifests/w/Wiimm/SZS/2.40a/Wiimm.SZS.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- InstallerUrl: https://szs.wiimm.de/download/szs-v2.40a-r8917-cygwin64.zip
+  Architecture: x64
+  NestedInstallerFiles:
+  - RelativeFilePath: szs-v2.40a-r8917-cygwin64\windows-install.exe
+  InstallerSha256: C54738CFCDAD74AE49424D4C3F8F16030290045632522263E22E8D7CD6D6EC76
+- InstallerUrl: https://szs.wiimm.de/download/szs-v2.40a-r8917-cygwin32.zip
+  Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: szs-v2.40a-r8917-cygwin32\windows-install.exe
+  InstallerSha256: B1EC14E3295693C167AD0859E75DD14ED1B71AAE71334EEF843A6C59D4783BF7
```
Comment (stephengillie) [inline manifests/w/Wiimm/SZS/2.40a/Wiimm.SZS.installer.yaml:18]:
```suggestion
  InstallerSha256: C54738CFCDAD74AE49424D4C3F8F16030290045632522263E22E8D7CD6D6EC76
```

Comment (stephengillie) [reply manifests/w/Wiimm/SZS/2.40a/Wiimm.SZS.installer.yaml:18]: I think they're the same file so need the same hash.

**Comment by stephengillie** [comment]
> @ stephengillie I don't know what to think about it

I feel the same. Going to run again to be sure.

**Comment by Exorcism0666** [comment]
same problem yeah @stephengillie

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335140 2024-01-03 11:52:39.141 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Archive.cpp(33)\WindowsPackageManager.dll!00007FF952A3413B: (caller: 00007FF9529AEBB5) ReturnHr(1) tid(aa8) 80004005 Unspecified error 2024-01-03 11:52:39.143 [CLI ] Failed to extract archive with code -2147467259 2024-01-03 11:52:46.819 [FAIL] D:\a\_work\1\s\external\pkg\src\AppInstallerCommonCore\Archive.cpp(33)\WindowsPackageManager.dll!77D7A99C: (caller: 77D0488A) ReturnHr(1) tid(b38) 80004005 Unspecified error 2024-01-03 11:52:46.820 [CLI ] Failed to extract archive with code -2147467259

(Automated response - build 515.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 3:57:54 PM: 2024-01-03 15:57:53.997 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Archive.cpp(33)\WindowsPackageManager.dll!00007FFB5A00209B: (caller: 00007FFB59F7DCCC) ReturnHr(1) tid(21bc) 80004005 Unspecified error 3:57:54 PM: 2024-01-03 15:57:53.997 [CLI ] Failed to extract archive with code -2147467259

(Automated response - build 513.)

**Comment by stephengillie** [comment]
Unfortunately, the ZIP file isn't valid. I tried to open it and it gave an error.

**Pull request closed without merging**